### PR TITLE
Addition of Costa Coffee

### PR DESCRIPTION
**Pull request by djrarky**

### CODE REVIEW

File: _data/food.yml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 websites:
+    - name: Costa Coffee
+      url: http://www.costa.com/
```
Comment (stephengroat) [inline _data/food.yml]: this is a landing page, i think you meant to use the `co.uk` tld

**Review by stephengroat — CHANGES_REQUESTED** [review]
Please correct the URL and describe how the site meets the criteria

**Comment by djrarky** [comment]
Done. Sorry about that.

And why wouldn't it match the criteria? It's a competitor to Starbucks with their own competing loyalty scheme.

**Comment by djrarky** [comment]
This might be helpful https://www.costa.co.uk/coffee-club/login :)

**Comment by stephengroat** [comment]
Starbucks has a payment scheme beyond the loyalty club

Site criteria is listed here (https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria)

**Comment by djrarky** [comment]
Costa stores personal profiles, names, addresses and tracks card usage including locations. [Looking at the exact form you sent me, that matches 1. and 2.](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria)

**Comment by stephengroat** [comment]
this site seems to have tended to stray away from listing loyalty programs. I'll let other in @2factorauth/collaborators comment on it

**Comment by djrarky** [comment]
well, I was only trying to help...

**Pull request closed without merging**